2022-04-10 21:21:26,843 - selector_events.py[line:58] - DEBUG: Using selector: KqueueSelector
2022-04-10 21:21:26,863 - server.py[line:75] - INFO: Started server process [37022]
2022-04-10 21:21:26,863 - on.py[line:45] - INFO: Waiting for application startup.
2022-04-10 21:21:26,863 - on.py[line:59] - INFO: Application startup complete.
2022-04-10 21:21:26,864 - server.py[line:211] - INFO: Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2022-04-10 21:21:45,657 - log.py[line:118] - INFO: SELECT DATABASE()
2022-04-10 21:21:45,658 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 21:21:45,674 - log.py[line:118] - INFO: SELECT @@sql_mode
2022-04-10 21:21:45,674 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 21:21:45,676 - log.py[line:118] - INFO: SELECT @@lower_case_table_names
2022-04-10 21:21:45,676 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 21:21:45,680 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 21:21:45,693 - log.py[line:118] - INFO: INSERT INTO algorithm (model_name, create_time, update_time) VALUES (%(model_name)s, %(create_time)s, %(update_time)s)
2022-04-10 21:21:45,694 - log.py[line:118] - INFO: [generated in 0.00158s] {'model_name': 'CNN', 'create_time': datetime.datetime(2022, 4, 10, 13, 21, 45, 693344), 'update_time': datetime.datetime(2022, 4, 10, 13, 21, 45, 693364)}
2022-04-10 21:21:45,785 - log.py[line:118] - INFO: ROLLBACK
2022-04-10 21:21:45,810 - h11_impl.py[line:369] - ERROR: Exception in ASGI application
Traceback (most recent call last):
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1820, in _execute_context
    cursor, statement, parameters, context
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 732, in do_execute
    cursor.execute(statement, parameters)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/cursors.py", line 148, in execute
    result = self._query(query)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/cursors.py", line 310, in _query
    conn.query(q)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/connections.py", line 548, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/connections.py", line 775, in _read_query_result
    result.read()
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/connections.py", line 1156, in read
    first_packet = self.connection._read_packet()
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1146, "Table 'raypipe.algorithm' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/uvicorn/protocols/http/h11_impl.py", line 366, in run_asgi
    result = await app(self.scope, self.receive, self.send)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/uvicorn/middleware/proxy_headers.py", line 75, in __call__
    return await self.app(scope, receive, send)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/fastapi/applications.py", line 261, in __call__
    await super().__call__(scope, receive, send)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/starlette/applications.py", line 112, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/starlette/middleware/errors.py", line 181, in __call__
    raise exc
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/starlette/middleware/errors.py", line 159, in __call__
    await self.app(scope, receive, _send)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/starlette/exceptions.py", line 82, in __call__
    raise exc
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/starlette/exceptions.py", line 71, in __call__
    await self.app(scope, receive, sender)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/fastapi/middleware/asyncexitstack.py", line 21, in __call__
    raise e
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/fastapi/middleware/asyncexitstack.py", line 18, in __call__
    await self.app(scope, receive, send)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/starlette/routing.py", line 656, in __call__
    await route.handle(scope, receive, send)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/starlette/routing.py", line 259, in handle
    await self.app(scope, receive, send)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/starlette/routing.py", line 61, in app
    response = await func(request)
  File "/Users/4paradigm/Desktop/hyperC/ocr/raypipe/raypipe/applications/app.py", line 44, in custom_route_handler
    return await original_route_handler(request)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/fastapi/routing.py", line 228, in app
    dependant=dependant, values=values, is_coroutine=is_coroutine
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/fastapi/routing.py", line 162, in run_endpoint_function
    return await run_in_threadpool(dependant.call, **values)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/starlette/concurrency.py", line 39, in run_in_threadpool
    return await anyio.to_thread.run_sync(func, *args)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/anyio/to_thread.py", line 29, in run_sync
    limiter=limiter)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/anyio/_backends/_asyncio.py", line 818, in run_sync_in_worker_thread
    return await future
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/anyio/_backends/_asyncio.py", line 754, in run
    result = context.run(func, *args)
  File "/Users/4paradigm/Desktop/hyperC/ocr/raypipe/raypipe/applications/sql/endpoints.py", line 24, in add_algorithm
    return crud.db_create_algo(db=db,algo=algo)
  File "/Users/4paradigm/Desktop/hyperC/ocr/raypipe/raypipe/applications/sql/crud.py", line 25, in db_create_algo
    db.commit()  # 提交保存到数据库中
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 1431, in commit
    self._transaction.commit(_to_root=self.future)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 829, in commit
    self._prepare_impl()
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 808, in _prepare_impl
    self.session.flush()
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 3363, in flush
    self._flush(objects)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 3503, in _flush
    transaction.rollback(_capture_exception=True)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py", line 72, in __exit__
    with_traceback=exc_tb,
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
    raise exception
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/session.py", line 3463, in _flush
    flush_context.execute()
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 456, in execute
    rec.execute(self)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 633, in execute
    uow,
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 250, in save_obj
    insert,
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 1241, in _emit_insert_statements
    execution_options=execution_options,
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1631, in _execute_20
    return meth(self, args_10style, kwargs_10style, execution_options)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 326, in _execute_on_connection
    self, multiparams, params, execution_options
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1508, in _execute_clauseelement
    cache_hit=cache_hit,
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1863, in _execute_context
    e, statement, parameters, cursor, context
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 2044, in _handle_dbapi_exception
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 207, in raise_
    raise exception
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1820, in _execute_context
    cursor, statement, parameters, context
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 732, in do_execute
    cursor.execute(statement, parameters)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/cursors.py", line 148, in execute
    result = self._query(query)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/cursors.py", line 310, in _query
    conn.query(q)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/connections.py", line 548, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/connections.py", line 775, in _read_query_result
    result.read()
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/connections.py", line 1156, in read
    first_packet = self.connection._read_packet()
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/Users/4paradigm/opt/miniconda3/envs/p37/lib/python3.7/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
sqlalchemy.exc.ProgrammingError: (pymysql.err.ProgrammingError) (1146, "Table 'raypipe.algorithm' doesn't exist")
[SQL: INSERT INTO algorithm (model_name, create_time, update_time) VALUES (%(model_name)s, %(create_time)s, %(update_time)s)]
[parameters: {'model_name': 'CNN', 'create_time': datetime.datetime(2022, 4, 10, 13, 21, 45, 693344), 'update_time': datetime.datetime(2022, 4, 10, 13, 21, 45, 693364)}]
(Background on this error at: https://sqlalche.me/e/14/f405)
2022-04-10 21:22:49,160 - server.py[line:252] - INFO: Shutting down
2022-04-10 21:22:49,265 - on.py[line:64] - INFO: Waiting for application shutdown.
2022-04-10 21:22:49,266 - on.py[line:75] - INFO: Application shutdown complete.
2022-04-10 21:22:49,266 - server.py[line:85] - INFO: Finished server process [37022]
2022-04-10 22:12:35,137 - log.py[line:118] - INFO: SELECT DATABASE()
2022-04-10 22:12:35,139 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:12:35,141 - log.py[line:118] - INFO: SELECT @@sql_mode
2022-04-10 22:12:35,141 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:12:35,142 - log.py[line:118] - INFO: SELECT @@lower_case_table_names
2022-04-10 22:12:35,142 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:12:35,144 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 22:12:35,145 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:12:35,145 - log.py[line:118] - INFO: [generated in 0.00029s] {'table_schema': 'raypipe', 'table_name': 'algorithm'}
2022-04-10 22:12:35,326 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:12:35,326 - log.py[line:118] - INFO: [cached since 0.1808s ago] {'table_schema': 'raypipe', 'table_name': 'experiment'}
2022-04-10 22:12:35,327 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:12:35,327 - log.py[line:118] - INFO: [cached since 0.1824s ago] {'table_schema': 'raypipe', 'table_name': 'model'}
2022-04-10 22:12:35,329 - log.py[line:118] - INFO: 
CREATE TABLE algorithm (
	id INTEGER NOT NULL AUTO_INCREMENT, 
	model_name VARCHAR(32), 
	create_time DATETIME NOT NULL, 
	update_time DATETIME NOT NULL, 
	PRIMARY KEY (id)
)


2022-04-10 22:12:35,329 - log.py[line:118] - INFO: [no key 0.00013s] {}
2022-04-10 22:12:35,440 - log.py[line:118] - INFO: CREATE INDEX ix_algorithm_model_name ON algorithm (model_name)
2022-04-10 22:12:35,440 - log.py[line:118] - INFO: [no key 0.00028s] {}
2022-04-10 22:12:35,481 - log.py[line:118] - INFO: CREATE INDEX ix_algorithm_id ON algorithm (id)
2022-04-10 22:12:35,481 - log.py[line:118] - INFO: [no key 0.00027s] {}
2022-04-10 22:12:35,497 - log.py[line:118] - INFO: 
CREATE TABLE experiment (
	id INTEGER NOT NULL AUTO_INCREMENT, 
	algorithm_id INTEGER, 
	train_params VARCHAR(256), 
	create_time DATETIME NOT NULL, 
	update_time DATETIME NOT NULL, 
	PRIMARY KEY (id)
)


2022-04-10 22:12:35,497 - log.py[line:118] - INFO: [no key 0.00050s] {}
2022-04-10 22:12:35,510 - log.py[line:118] - INFO: CREATE INDEX ix_experiment_algorithm_id ON experiment (algorithm_id)
2022-04-10 22:12:35,510 - log.py[line:118] - INFO: [no key 0.00020s] {}
2022-04-10 22:12:35,521 - log.py[line:118] - INFO: CREATE INDEX ix_experiment_id ON experiment (id)
2022-04-10 22:12:35,521 - log.py[line:118] - INFO: [no key 0.00021s] {}
2022-04-10 22:12:35,529 - log.py[line:118] - INFO: 
CREATE TABLE model (
	id INTEGER NOT NULL AUTO_INCREMENT, 
	exp_id INTEGER, 
	save_path VARCHAR(256), 
	create_time DATETIME NOT NULL, 
	update_time DATETIME NOT NULL, 
	PRIMARY KEY (id)
)


2022-04-10 22:12:35,529 - log.py[line:118] - INFO: [no key 0.00042s] {}
2022-04-10 22:12:35,543 - log.py[line:118] - INFO: CREATE INDEX ix_model_id ON model (id)
2022-04-10 22:12:35,543 - log.py[line:118] - INFO: [no key 0.00018s] {}
2022-04-10 22:12:35,558 - log.py[line:118] - INFO: CREATE INDEX ix_model_exp_id ON model (exp_id)
2022-04-10 22:12:35,558 - log.py[line:118] - INFO: [no key 0.00038s] {}
2022-04-10 22:12:35,572 - log.py[line:118] - INFO: COMMIT
2022-04-10 22:12:35,578 - selector_events.py[line:58] - DEBUG: Using selector: KqueueSelector
2022-04-10 22:12:35,629 - server.py[line:75] - INFO: Started server process [42652]
2022-04-10 22:12:35,630 - on.py[line:45] - INFO: Waiting for application startup.
2022-04-10 22:12:35,630 - on.py[line:59] - INFO: Application startup complete.
2022-04-10 22:12:35,633 - server.py[line:211] - INFO: Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2022-04-10 22:12:41,614 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 22:12:41,616 - log.py[line:118] - INFO: INSERT INTO algorithm (model_name, create_time, update_time) VALUES (%(model_name)s, %(create_time)s, %(update_time)s)
2022-04-10 22:12:41,616 - log.py[line:118] - INFO: [generated in 0.00035s] {'model_name': 'CNN', 'create_time': datetime.datetime(2022, 4, 10, 14, 12, 41, 615985), 'update_time': datetime.datetime(2022, 4, 10, 14, 12, 41, 615997)}
2022-04-10 22:12:41,620 - log.py[line:118] - INFO: COMMIT
2022-04-10 22:12:41,623 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 22:12:41,630 - log.py[line:118] - INFO: SELECT algorithm.id, algorithm.model_name, algorithm.create_time, algorithm.update_time 
FROM algorithm 
WHERE algorithm.id = %(pk_1)s
2022-04-10 22:12:41,630 - log.py[line:118] - INFO: [generated in 0.00027s] {'pk_1': 1}
2022-04-10 22:12:41,634 - log.py[line:118] - INFO: ROLLBACK
2022-04-10 22:13:04,820 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 22:13:04,821 - log.py[line:118] - INFO: SELECT algorithm.id AS algorithm_id, algorithm.model_name AS algorithm_model_name, algorithm.create_time AS algorithm_create_time, algorithm.update_time AS algorithm_update_time 
FROM algorithm
2022-04-10 22:13:04,821 - log.py[line:118] - INFO: [generated in 0.00027s] {}
2022-04-10 22:13:21,159 - log.py[line:118] - INFO: SELECT DATABASE()
2022-04-10 22:13:21,159 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:13:21,160 - log.py[line:118] - INFO: SELECT @@sql_mode
2022-04-10 22:13:21,160 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:13:21,160 - log.py[line:118] - INFO: SELECT @@lower_case_table_names
2022-04-10 22:13:21,160 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:13:21,161 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 22:13:21,162 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:13:21,162 - log.py[line:118] - INFO: [generated in 0.00015s] {'table_schema': 'raypipe', 'table_name': 'algorithm'}
2022-04-10 22:13:21,168 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:13:21,168 - log.py[line:118] - INFO: [cached since 0.006743s ago] {'table_schema': 'raypipe', 'table_name': 'experiment'}
2022-04-10 22:13:21,170 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:13:21,170 - log.py[line:118] - INFO: [cached since 0.008042s ago] {'table_schema': 'raypipe', 'table_name': 'model'}
2022-04-10 22:13:21,171 - log.py[line:118] - INFO: COMMIT
2022-04-10 22:13:21,174 - selector_events.py[line:58] - DEBUG: Using selector: KqueueSelector
2022-04-10 22:13:21,195 - server.py[line:75] - INFO: Started server process [42856]
2022-04-10 22:13:21,195 - on.py[line:45] - INFO: Waiting for application startup.
2022-04-10 22:13:21,195 - on.py[line:59] - INFO: Application startup complete.
2022-04-10 22:13:21,196 - server.py[line:211] - INFO: Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2022-04-10 22:13:24,437 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 22:13:24,441 - log.py[line:118] - INFO: SELECT algorithm.id AS algorithm_id, algorithm.model_name AS algorithm_model_name, algorithm.create_time AS algorithm_create_time, algorithm.update_time AS algorithm_update_time 
FROM algorithm
2022-04-10 22:13:24,441 - log.py[line:118] - INFO: [generated in 0.00036s] {}
2022-04-10 22:14:10,064 - log.py[line:118] - INFO: SELECT DATABASE()
2022-04-10 22:14:10,064 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:14:10,066 - log.py[line:118] - INFO: SELECT @@sql_mode
2022-04-10 22:14:10,066 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:14:10,067 - log.py[line:118] - INFO: SELECT @@lower_case_table_names
2022-04-10 22:14:10,067 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:14:10,068 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 22:14:10,070 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:14:10,070 - log.py[line:118] - INFO: [generated in 0.00031s] {'table_schema': 'raypipe', 'table_name': 'algorithm'}
2022-04-10 22:14:10,074 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:14:10,075 - log.py[line:118] - INFO: [cached since 0.005161s ago] {'table_schema': 'raypipe', 'table_name': 'experiment'}
2022-04-10 22:14:10,076 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:14:10,077 - log.py[line:118] - INFO: [cached since 0.007081s ago] {'table_schema': 'raypipe', 'table_name': 'model'}
2022-04-10 22:14:10,078 - log.py[line:118] - INFO: COMMIT
2022-04-10 22:14:10,085 - selector_events.py[line:58] - DEBUG: Using selector: KqueueSelector
2022-04-10 22:14:10,135 - server.py[line:75] - INFO: Started server process [42952]
2022-04-10 22:14:10,135 - on.py[line:45] - INFO: Waiting for application startup.
2022-04-10 22:14:10,136 - on.py[line:59] - INFO: Application startup complete.
2022-04-10 22:14:10,137 - server.py[line:211] - INFO: Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2022-04-10 22:15:01,083 - server.py[line:252] - INFO: Shutting down
2022-04-10 22:15:01,189 - server.py[line:270] - INFO: Waiting for connections to close. (CTRL+C to force quit)
2022-04-10 22:15:06,668 - log.py[line:118] - INFO: SELECT DATABASE()
2022-04-10 22:15:06,668 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:15:06,669 - log.py[line:118] - INFO: SELECT @@sql_mode
2022-04-10 22:15:06,670 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:15:06,670 - log.py[line:118] - INFO: SELECT @@lower_case_table_names
2022-04-10 22:15:06,671 - log.py[line:118] - INFO: [raw sql] {}
2022-04-10 22:15:06,672 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 22:15:06,674 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:15:06,674 - log.py[line:118] - INFO: [generated in 0.00029s] {'table_schema': 'raypipe', 'table_name': 'algorithm'}
2022-04-10 22:15:06,676 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:15:06,676 - log.py[line:118] - INFO: [cached since 0.002432s ago] {'table_schema': 'raypipe', 'table_name': 'experiment'}
2022-04-10 22:15:06,677 - log.py[line:118] - INFO: SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = %(table_schema)s AND table_name = %(table_name)s
2022-04-10 22:15:06,677 - log.py[line:118] - INFO: [cached since 0.003948s ago] {'table_schema': 'raypipe', 'table_name': 'model'}
2022-04-10 22:15:06,679 - log.py[line:118] - INFO: COMMIT
2022-04-10 22:15:06,685 - selector_events.py[line:58] - DEBUG: Using selector: KqueueSelector
2022-04-10 22:15:06,728 - server.py[line:75] - INFO: Started server process [43054]
2022-04-10 22:15:06,729 - on.py[line:45] - INFO: Waiting for application startup.
2022-04-10 22:15:06,729 - on.py[line:59] - INFO: Application startup complete.
2022-04-10 22:15:06,730 - server.py[line:211] - INFO: Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2022-04-10 22:15:12,220 - log.py[line:118] - INFO: BEGIN (implicit)
2022-04-10 22:15:12,228 - log.py[line:118] - INFO: SELECT algorithm.id AS algorithm_id, algorithm.model_name AS algorithm_model_name, algorithm.create_time AS algorithm_create_time, algorithm.update_time AS algorithm_update_time 
FROM algorithm
2022-04-10 22:15:12,228 - log.py[line:118] - INFO: [generated in 0.00075s] {}
2022-04-10 22:15:12,255 - log.py[line:118] - INFO: ROLLBACK
